mmtrack/models/mot/ocsort.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import torch
+from mmdet.models import build_detector
+
+from mmtrack.core import outs2results, results2outs
+from ..builder import MODELS, build_motion, build_tracker
+from .base import BaseMultiObjectTracker
+
+
+@MODELS.register_module()
+class OCSORT(BaseMultiObjectTracker):
```

Comment:
You can inherit this class from `DeepSORT`?

Reply:
Why? I follow the convention of `ByteTrack` to use `BaseMultiObjectTracker` as they don't use appearance.